main/autogen/data/msgpack_lite.cc

```diff
@@ -184,6 +189,7 @@ const map<int, vector<string>> MsgpackWriterLite::parsedFileAttrMap{{
                                                                     {
                                                                         7,
                                                                         {
+                                                                            "typed_level",
```

Comment:
ooc, why is it not also called `strict_level` like it would be in Sorbet?

Reply:
Historical baggage, I guess -- the non-lite version called it `typed_level`, and we have tests that require that the lite fields are a subset of the non-lite fields and exist in the same relative order.

I think we could get away with changing the name here in the future.